### PR TITLE
Bump version number to 1.5.7-dev.0

### PR DESCRIPTION
**Pull request by hoechenberger**
We should automate this…

x-ref GH-1080

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -1,7 +1,4 @@
 {
-  "packages": [
-    "bids-validator",
-    "bids-validator-web"
-  ],
-  "version": "1.5.6"
+  "packages": ["bids-validator", "bids-validator-web"],
```
Comment (hoechenberger) [inline lerna.json:2]: formatting change seems to be due to a new pre-commit hook? I didn't add this myself…

**Comment by sappelhoff** [comment]
realizing that 1.5.6 was skipped and looking into the release notes shows that there was some issue with the release process --- perhaps due to our usage of `dev` we should wait with merging this PR until @rwblair has had a look

**Comment by hoechenberger** [comment]
> realizing that 1.5.6 was skipped and looking into the release notes shows that there was some issue with the release process --- perhaps due to our usage of `dev`

I don't think that's related though? 1.5.5 was skipped, and we only added our `-dev.0` change in the 1.5.6 cycle. But yes let's wait for feedback.

**Comment by sappelhoff** [comment]
mmh true... no idea.

> We should automate this…

do you have a proposal?

**Comment by hoechenberger** [comment]
> > We should automate this…
> 
> do you have a proposal?

Nope :)
We could also just add this to the "how to make a release" docs :) It really just takes a minute to do, so… not sure if it's even worth trying to automate it.

**Comment by rwblair** [comment]
My thinking yesterday was that I would bump the version in the PR that would cause the latest commit on master to actually differ from latest tagged release. I'll spend some time today seeing if this can just be incorporated into the lerna publish workflow so that the repo only need to be touched once for two version changes.

**Comment by rwblair** [comment]
Ok added a script here a96790e that should help do this automatically when I publish. Well see how it works soon.

**Comment by sappelhoff** [comment]
nice, let's see how it'll work during next release :)

**Pull request merged**